docs/running-tests/testing-polyfills.md

```diff
@@ -0,0 +1,63 @@
+# Testing polyfills
+
+## Preparing the polyfill
+
+The polyfill script-injection feature currently only supports scripts which
+are immediately invoked. The script must be prepared as a single file whose
+contents will be inlined into a script tag served as part of every test page.
+
+If your polyfill is only available as an asynchronous module with dependent
```

Comment:
Oh, I didn't consider this problem. Do you think there are polyfills which need to do async stuff to initialize themselves? If so, having those polyfills reliably be loaded before tests run would be hard, we'd need some mechanism to delay tests running, but that itself could break tests written using synchronous `test()`...

Hopefully this isn't a real-world problem?

Reply:
Good! I wasn't sure if this comment was gesturing at such a problem or not.